ProjectUpdater/nbproject/build-impl.xml

```diff
@@ -0,0 +1,1411 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be committed.

Reply:
I think this file is fine @cygnus-x-1, we do it for all of our modules too. It's not in the "nbproject/private" folder which is in our `.gitignore`. Please confirm your OK?

Reply:
fair enough... counterpoint, neither is genfiles.properties.